### DISCUSSION TITLE
Copy changes back from xcode proj back into main project workspace

### DISCUSSION THREAD

**Discussion by jcroucher** [ionic-team/capacitor#5169] [Q&A] [answered]
Is there a correct workflow for copying changes back to the main project from xcode?

Every time I do the build and sync step, it updates the xcode project. Xcode then gives me a warning saying that I need to update the project settings before I can build. 
I select this, it does its thing then I can build.

Is there a way to copy these changes back so they are remembered on each new build?

**Comment by jcesarmobile** [comment, marked as answer]
What project are you referring to? Capacitor doesn’t do any changes on the users projects, and users should no make changes on capacitor’s projects

**Reply by jcroucher** [reply]
When you run npx cap sync it rebuilds ios/App/Pods/Pods.xcodeproj
Then when I open App.xcodeproj in xcode and try to build it gives a warning "Validate project settings" -- "Update to recommended settings"
If I ignore this, the build fails.
Clicking it brings up a dialog showing all the settings xcode is going to change. I accept the changes then it builds fine, until I run npx cap sync again which then the project settings are invalid again.

**Reply by jcesarmobile** [reply]
You should never change the files in Pods folder, would be good if you could provide a sample app with the problem so it can be fixed. Every Xcode major update tells you to update settings, but that's just a warning and I can build without changing the settings, so your problem should be something else.

**Reply by jcroucher** [reply]
You are right. One of the plugins had a deployment version of 11 instead of 12. 
Updating the settings was fixing this error. I manually updated the plugin target version. 

It started being an issue when running the latest xcode beta.
